src/screenshot/screenshot-compare.ts

```diff
@@ -104,15 +104,11 @@ export async function compareScreenshot(
       // compare the two images pixel by pixel to
       // figure out a mismatch value
 
-      // figure out the actual width and height of the screenshot
-      const naturalWidth = Math.round(emulateConfig.viewport.width * emulateConfig.viewport.deviceScaleFactor);
-      const naturalHeight = Math.round(emulateConfig.viewport.height * emulateConfig.viewport.deviceScaleFactor);
-
       const pixelMatchInput: d.PixelMatchInput = {
         imageAPath: join(screenshotBuildData.imagesDir, screenshot.diff.imageA),
         imageBPath: join(screenshotBuildData.imagesDir, screenshot.diff.imageB),
-        width: naturalWidth,
-        height: naturalHeight,
+        width,
+        height,
```

Comment:
GH's having some issues this morning, the first time I submitted something I got a 5xx error. I don't think the original went through, but if it did, apologies if this is a duplicate review!

Previously, we were rounding out the values to the nearest integer for this value - for my own understanding:
1. Does it matter if floats can be provided here?
2. If so, do we have any guarantee the values of `width` or `height` are integers?

Reply:
> 1. Does it matter if floats can be provided here?

I don't believe so, let me verify that this doesn't cause an issue.

> 2\. If so, do we have any guarantee the values of `width` or `height` are integers?

Since this is user input we can't guarantee the values to be integer. Let me verify if using float values has an impact on pixelmatch and if so add a warning or error out in case a float value was provided.

Reply:
I was able to verify that a rounding operation is done when storing the images causing a failure when comparing the image sizes with float values. I pushed a patch to round values and everything works as expected. Thanks!